routes/main.go

```diff
@@ -2,31 +2,31 @@ package routes
 
 import (
 	"os"
-	"strings"
 
 	"github.com/gin-contrib/cors"
 	"github.com/gin-gonic/gin"
 	"github.com/uptrace/bun"
 )
 
 func SetupV1Routes(db *bun.DB) *gin.Engine {
-	var router = gin.Default()
+	router := gin.Default()
+
+	allowedOrigins := os.Getenv("ALLOWED_ORIGINS")
+
+	corsConfig := cors.Config{
+		AllowOrigins:     []string{allowedOrigins},
+		AllowMethods:     []string{"GET", "POST"},
```

Comment:
are we not supporting PATCH and DELETE

Reply:
I have added thanks for pointing this out